src/vivarium_gates_mncnh/components/observers.py

```diff
@@ -199,6 +231,13 @@ def map_acs_eligibility(self, pop: pd.DataFrame) -> pd.Series:
         is_eligible = pop[COLUMNS.STATED_GESTATIONAL_AGE].between(26, 33)
         return is_eligible.rename("acs_eligibility")
 
+    def map_true_hemoglobin(self, pop: pd.DataFrame) -> pd.Series:
+        exposure = pop[PIPELINES.IFA_DELETED_HEMOGLOBIN_EXPOSURE]
```

Comment:
This should be after the effect of IFA/MMS is applied, not the IFA deleted value